docs/docs/cmd/spe/containertype/containertype-list.mdx

```diff
@@ -0,0 +1,130 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Container Types list
+
+Retrieves a list of Container Types created for a SharePoint Embedded Application
+
+## Usage
+
+```sh
+m365 Container Types list [options]
+```
+
+## Options
+
+<Global />
+
+## Examples
+
+Retrieves a list of Container Types created for a SharePoint Embedded Application from the tenant.
+
+```sh
+m365 Container Types list
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  [{
+    "_ObjectType_": "Microsoft.Online.SharePoint.TenantAdministration.SPContainerTypeProperties",
+    "ApplicationRedirectUrl": null,
+    "AzureSubscriptionId": "/Guid(00000000-0000-0000-0000-000000000000)/",
+    "ContainerTypeId": "/Guid(073269af-f1d2-042d-2ef5-5bdd6ac83115)/",
+    "CreationDate": null,
+    "DisplayName": "test1",
+    "ExpiryDate": null,
+    "IsBillingProfileRequired": true,
+    "OwningAppId": "/Guid(df4085cc-9a38-4255-badc-5c5225610475)/",
+    "OwningTenantId": "/Guid(00000000-0000-0000-0000-000000000000)/",
+    "Region": null,
+    "ResourceGroup": null,
+    "SPContainerTypeBillingClassification": 0
+  },
+   {
+    "_ObjectType_": "Microsoft.Online.SharePoint.TenantAdministration.SPContainerTypeProperties",
+    "ApplicationRedirectUrl": null,
+    "AzureSubscriptionId": "/Guid(00000000-0000-0000-0000-000000000000)/",
+    "ContainerTypeId": "/Guid(3ec7c59d-ef31-0752-1ab5-5c343a5e8557)/",
+    "CreationDate": null,
+    "DisplayName": "SharePoint Embedded Free Trial Container Type",
+    "ExpiryDate": null,
+    "IsBillingProfileRequired": true,
+    "OwningAppId": "/Guid(e6b529fe-84db-412a-8e9c-5bbb75f7b037)/",
+    "OwningTenantId": "/Guid(00000000-0000-0000-0000-000000000000)/",
+    "Region": null,
+    "ResourceGroup": null,
+    "SPContainerTypeBillingClassification": 1
+  }]
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  ContainerTypeId                             
+  --------------------------------------------
+  /Guid(880ab3bd-5b68-01d4-3744-01a7656cf2ba)/
+  /Guid(3ec7c59d-ef31-0752-1ab5-5c343a5e8557)/  
+  ```
```

Comment:
in the `defaultProperties` method besides the id we also return the Container Type Name and Owner App Id. The problem is we specified wrong column names that's why you did not get them in the response. but if we correct those names we will get 3 column for the `text` `output` mode
lets be sure to update it 👍

Reply:
Thanks for spotting, really sorry for getting the columns wrong

Reply:
no problem 🙂